api/v1alpha4/machine_types.go

```diff
@@ -210,7 +210,7 @@ func (m *MachineStatus) GetTypedPhase() MachinePhase {
 
 // ANCHOR: Bootstrap
 
-// Bootstrap capsulates fields to configure the Machine’s bootstrapping mechanism.
+// Bootstrap encapsulates fields to configure the Machine’s bootstrapping mechanism.
```

Comment:
I think you need to run `make generate` to actually update the description in config/

Reply:
Uhmm this should be captured by verify scripts...

Reply:
I don't think the struct comment is actually used anywhere, the description of the field (when using a struct as a pointer or not) is derived from the comment on that field, not the one on the linked struct. 

Tested the verify script locally and it's still working properly.